.gitignore

```diff
@@ -32,6 +32,7 @@ modules.builtin
 *.lzo
 *.patch
 *.gcno
+out/
```

Comment:
While I'd like to get an out folder, this should not be in gitignore, as building with outfolder is not a default kernel compile "option".